frontend/src/components/pages/rp-connect/Pipelines.Create.tsx

```diff
@@ -194,12 +200,46 @@ class RpConnectPipelinesCreate extends PageComponent<{}> {
 
 export default RpConnectPipelinesCreate;
 
+interface QuickActions {
+  editorInstance: editor.IStandaloneCodeEditor | null;
+}
+
+const QuickActions = ({ editorInstance }: QuickActions) => {
+  const { isOpen: isAddSecretOpen, onOpen: openAddSecret, onClose: closeAddSecret } = useDisclosure();
+
+  if (editorInstance === null) {
+    return <Box minW={300} />;
+  }
```

Comment:
Why not some Empty state?

Reply:
Because if we don't get the editor instance, any of these operations can be applied, and also this case should have no impact on the customer experience, this case is not a break.